cognite/client/data_classes/transformations/__init__.py

```diff
@@ -49,6 +49,17 @@ def __init__(
         self.client_id = client_id
         self.project_name = project_name
 
+    @classmethod
+    def _load(cls, resource: dict[str, Any]) -> SessionDetails:
+        to_load = {}
+        if "sessionId" in resource:
+            to_load["session_id"] = resource["sessionId"]
+        if "clientId" in resource:
+            to_load["client_id"] = resource["clientId"]
+        if "projectName" in resource:
+            to_load["project_name"] = resource["projectName"]
+        return cls(**to_load)
```

Comment:
Why not just?
```suggestion
        return cls(
            session_id=resource.get("sessionId"),
            client_id=resource.get("clientId"),
            project_name=resource.get("projectName"),
        )
```

Reply:
Why would we want to use default values for the parameters when loading the response from the API?

Reply:
All keys should be present in responses.

Reply:
Also thinking that none of these attributes should actually be nullable.

Reply:
Aight, will change. None were listed as required in the official API docs hence the checks